mne/decoding/time_gen.py

```diff
@@ -123,10 +123,16 @@ def fit(self, epochs, y=None):
         If X is a dense array, then the other methods will not support sparse
         matrices as input.
         """
-        from sklearn.base import clone
-        from sklearn.cross_validation import check_cv, StratifiedKFold
+        from mne.utils import check_version
```

Comment:
this should be a top level import is what @Eric89GXL is saying. A relative import